src/test/kotlin/org/rust/lang/core/resolve/RsStdlibResolveTest.kt

```diff
@@ -351,4 +351,27 @@ class RsStdlibResolveTest : RsResolveTestBase() {
         }
     """)
 
+    fun `test resolve derive traits`() {
+        val traitToPath = mapOf(
+            "Clone" to "clone.rs",
+            "Copy" to "marker.rs",
+            "Debug" to "fmt/mod.rs",
+            "Default" to "default.rs",
+            "Eq" to "cmp.rs",
+            "Hash" to "hash/mod.rs",
+            "Ord" to "cmp.rs",
+            "PartialEq" to "cmp.rs",
+            "PartialOrd" to "cmp.rs"
+        )
+        for ((trait, path) in traitToPath) {
+            // use different file names to avoid exception because of
+            // creation of already existing file
+            stubOnlyResolve("""
+            //- ${trait.toLowerCase()}.rs
```

Comment:
I don't think so, `stubOnlyResolve` is intended to be used as a single expression inside the test method body. I think the way to go is to deconstruct `stubOnlyResolve` into helper functions and use them here.